opencodelists/settings.py

```diff
@@ -30,29 +30,58 @@
 sys.modules["sqlite3"] = sys.modules.pop("sqlean")
 
 
-# Read env file/vars
-env = Env()
-env.read_env()
+_missing_env_var_hint = """\
+If you are running commands locally outside of `just` then you should
+make sure that your `.env` file is being loaded into the environment,
+which you can do in Bash using:
+
+    set -a; source .env; set +a
+
+If you are seeing this error when running via `just` (which should
+automatically load variables from `.env`) then you should check that
+`.env` contains all the variables listed in `dotenv-sample` (which may
+have been updated since `.env` was first created).
+
+If you are seeing this error in production then you haven't configured
+things properly.
+"""
+
+
+def get_env_var(name, **kwargs):
+    try:
+        env = os.environ[name]
+        if "default" in kwargs:
+            t = type(kwargs["default"])
+            if t is bool:
+                return env == "True"
+            return t(env)
+        return env
+    except KeyError:
+        if "default" in kwargs:
+            return kwargs["default"]
+        raise RuntimeError(
```

Comment:
Arguably `KeyError` is a better fit than `RuntimeError` if something is missing from the `os.environ` dict. I don't feel strongly though.

Reply:
I don't have feelings either way, this was lifted straight from the airlock code